### PR TITLE
Ops 2367/initial UI

### PR DESCRIPTION
**Pull request by rajohnson90**
## What changed

Added a document tab to Agreements. The document is currently populated with static data so every agreement has the same list of six documents. This commit contains no major logic for interacting with documents.

## Issue

[OPS-2367](https://app.zenhub.com/workspaces/opre-ops-62d197b4c468970013bcd2ec/issues/gh/hhs/opre-ops/2367)

## How to test

1. Log in as any account
2. Click Agreements in the header
3. Click on any agreement
4. Click on the documents tag

You should see 6 documents in 3 rows of 2. 

## Screenshots

image

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [ ] Automated integration tests updated and passed
- [ ] Automated quality tests updated and passed
- [ ] Automated load tests updated and passed
- [ ] Automated a11y tests updated and passed
- [ ] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

## Links

_If relevant, e.g. for a link to a piece of markdown documentation_

### CODE REVIEW

**Review by johndeange — APPROVED** [review]

**Review by maiyerlee — APPROVED** [review]
Kudos on the document page! It looks great and will be useful for testing the endpoint I'm working on. 🎆 🎇

**Review by fpigeonjr — APPROVED** [review]
⭐ Looks good @rajohnson90 

non-blocking suggestion...

Feel like the Documents components shouldn't live under pages...
- maybe they could live under `/src/components` or `/src/agreements` since they belong to Agreements (but maybe they will be part of other contract types so maybe components is better 😄 )
- thanks for adding unit-tests and JSDocs

File: frontend/src/components/UI/DocumentCard/DocumentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const uploaded_by_text = `Uploaded by ${document.uploaded_by} on ${document.upload_date}`;
+    return (
+        <div
+            className={cardContainer}
+            data-cy={dataCy ?? dataCy}
+            {...rest}
+        >
+            <div className="display-flex flex-column margin-left-4 margin-top-205">
+                <div className={documentTitleClassNames}>{document.title}</div>
+                <div className="font-sans-3xs text-bold margin-top-05">{document.filename}</div>
```
Comment (fpigeonjr) [inline frontend/src/components/UI/DocumentCard/DocumentCard.jsx:27]: to get the 12px font-size we came up with a custom classname called 'font-12px' 😄

Comment (rajohnson90) [reply frontend/src/components/UI/DocumentCard/DocumentCard.jsx:27]: Thanks, I did not know this. Also I had similar feelings about potentially moving the documents into components as well. I will do that.

**Pull request merged**